tests/Fixtures/e2e/Config_Framework/infection.json

```diff
@@ -0,0 +1,14 @@
+{
+    "timeout": 25,
+    "source": {
+        "directories": [
+            "src"
+        ]
+    },
+    "logs": {
+        "debug": "infection-log.txt"
+    },
+    "system": {
```

Comment:
I'm not sure we need a `system` key. It's ambiguous. Why `tmpDir` is on the root level, but `bootstrap` is under `system`? https://infection.github.io/guide/usage.html#Configuration

I suggest add `bootstrap` and `testFramework` to the root level to simplify json

Reply:
Personally, I think the opposite is more appropriate - move tmpDir (and others) to the system entry. In that way all the Infection specific settings are grouped together away from the testing framework configurations. Additionally, it might be better to name it 'config' instead of 'system' to follow the naming convention of composer, but that's neither here nor there.

Reply:
After updating the site repo, changed my mind. I've converted it over to root.